test/gui.lua

```diff
@@ -1,3 +1,5 @@
+config.targets = 'core'
```

Comment:
Are we planning to support multiple targets per test? Would suggest `target` if not.

Reply:
I had the quickfort suite of integration tests in mind when I made this plural. I could also make a singular variant of the property. The overwhelming majority of tests test one target (being "unit" tests and all). Maybe calling it "target" but still letting it take a list for integration cases is the way to go.